app/views/paginable/plans/_privately_visible.html.erb

```diff
@@ -16,7 +16,7 @@
           </thead>
           <tbody>
             <% scope.each do |plan| %>
-              <tr>
+              <tr id="<%= dom_id(plan) %>">
```

Comment:
It causes no harm, just curious why you would introduce an id attribute here. I don't see any JS that refers to it.

Reply:
It's used in the specs to make sure we click the correct link.